api-report/sequence.api.md

```diff
@@ -136,6 +136,11 @@ export interface IIntervalCollection<TInterval extends ISerializableInterval> ex
     gatherIterationResults(results: TInterval[], iteratesForward: boolean, start?: number, end?: number): void;
     // (undocumented)
     getIntervalById(id: string): TInterval | undefined;
+    // (undocumented)
+    getSlideToSegment(lref: LocalReferencePosition): {
```

Comment:
thinking about how to avoid exposing this on the interface. one option that may be possible is to covert _getSlideToSegment in merge-tree to a free function, and export it. then anyone with a segment can find where it would side using that, and it doesn't need to be at this api layer.

Reply:
we would also deprecate getSlideToSegment on client, as consumers could just move to the free function. doing something like this would probably be better as a separate PR